apps/src/lib/tools/jsdebugger/DebugConsole.jsx

```diff
@@ -0,0 +1,187 @@
+import React, {PropTypes} from 'react';
```

Comment:
Please add a file level comment with at least as much info as the one in JsDebugger.jsx